src/installer/corehost/cli/hostmisc/pal.unix.cpp

```diff
@@ -14,6 +14,7 @@
 #include <fnmatch.h>
 #include <ctime>
 #include <pwd.h>
+#include <config.h>
```

Comment:
```suggestion
#include "config.h"
```

Reply:
Any particular reason for not keeping it with diamond brackets?

Reply:
some related stats:

```sh
> git grep "\"config.h\"" | wc -l
     162
> git grep "<config.h>" | wc -l
     314
```

Reply:
It's all in Mono, as you can see -- in CoreCLR, the quotes are used.